string                              id
string                              agent_name
bool                                acknowledged
string[]               items
